cmd/rook/ceph/mgr.go

```diff
@@ -67,6 +69,15 @@ func init() {
 func runMgrSidecar(cmd *cobra.Command, args []string) error {
 	rook.SetLogLevel()
 
+	logger.Infof("runMgrSidecar get ExternalIps:%s", dashboardExternalIps)
+	if dashboardExternalIps != "" {
+		for _, ip := range strings.Split(dashboardExternalIps, ",") {
+			if ip != "" {
+				clusterSpec.Dashboard.ExternalIps = append(clusterSpec.Dashboard.ExternalIps, ip)
```

Comment:
You could just assign the results of `Split()` directly and no need for the loop, right?
```suggestion
		clusterSpec.Dashboard.ExternalIps = strings.Split(clusterSpec.Dashboard.ExternalIps, ",")
```